index.html

```diff
@@ -275,32 +275,32 @@ <h3 id="contact2">
         I'm always interested in hearing about new projects, so if you'd like
         to chat please get in touch.
       </h3>
-      <form action="https://formspree.io/f/mayvdjkb" method="POST" class="form apply-flex" id="contact-form">
+      <form action="https://formspree.io/f/xvoyekjg" method="post" class="form apply-flex" id="contact-form">
         
         <!-- Adding labels to all form elements for accessibility reasons and make them invisible by adding css rules display: none for the sake of Figma design" -->
-        <label for="user_fullname">Full Name</label>
-        <input type="text" name="user_fullname" placeholder="Full name" maxlength="30" />
+        <label for="name">Full Name</label>
+        <input type="text" name="name" id="name" placeholder="Full name" maxlength="30" required/>
```

Comment:
- [ ] There's a problem with the submission of your contact form. You have different input fields for the name on both the desktop and mobile versions hiding them depending on the type of device and marking them as required. This is causing your form to have unreachable fields preventing the correct submission.

Please consider refactoring your form, you only need one input field for the name (no need to separate it into First and Last names), and make sure that it is working as expected. 🤓 🛠️